packages/appstore/src/components/main-title-bar/index.tsx

```diff
@@ -38,9 +32,6 @@ const MainTitleBar = () => {
     return (
         <React.Fragment>
             <DesktopWrapper>
-                {/* TODO: This is for testing purposes only */}
```

Comment:
Why did you remove it?We need to remove it once we connect to server_mock or actual API

Reply:
@mahdiyeh-deriv In my opinion, it's just a button to test the modal. It's already tested by QA and I thought I can delete this button. Let me know your final decision and I'll return this button if we still need this.